material/problems/Kinematics/SwingingShip/SwingingShip.tex

```diff
@@ -24,48 +24,110 @@
+      the ship in contact with the tire, $\vel{B_A}{N}$.
       %
+    \\[0.45pc]
+    \Solution{}{0.99\linewidth}{
+      \begin{tabular}{@{}l@{ \equals[\;] }l}
+        $\vel{A_B}{N}$
+        & $\vel{A_o}{N}\plus{}^N\vec{\omega}^A \times\vec{r}^{A_B\slash A_o}$
+        \\[0.5pc]
+        &$\vec{0} \plus(\omega_A \hat{n_z}) \times(R_A \hat{n_y})$
+        \\[0.5pc]
+        &$\omega_AR_A(\minus\hat{n_x})$
+      \end{tabular}
+      \hspace{1cm}
+      \begin{tabular}{@{}l@{ \equals[\;] }l}
+        $\vel{B_A}{N}$
+        & $\vel{B_o}{N}\plus{}^N\vec{\omega}^B \times\vec{r}^{B_A \slash B_o}$
+        \\[0.5pc]
+        &$\vec{0} \plus (\omega_B \hat{n_z}) \times (\minus R_B \hat{n_y})$
+        \\[0.5pc]
+        &$\omega_B R_B(\hat{n_x})$
+      \end{tabular}
+      \\[2.0pc]
+    }
+	    %
     \item When the ship \textbf{rolls} on the tire, write the relevant
       \textbf{constraint equation} and use it to find a \textbf{scalar}
       relationship between $\omega_A$ and $\omega_B$.
       %
+    \\[0.45pc]
+    \Solution{}{0.99\linewidth}{
+      \begin{tabular}{l@{}r@{\equals[\;]}l}
+        Rolling Constraint:
+        &$\vel{A_B}{N}$
+        &$\vel{B_A}{N}$
+        \\[0.5pc]
+        &$\minus \omega_A R_A \hat{n_x}$
+        &$\omega_B R_B \hat{n_x}$
+        \\[0.5pc]
+        dot with $\hat{n_x}$:
+        &$\minus \omega_A R_A$
+        &$\omega_B R_B$
+      \end{tabular}}
+      %
     \item \textbf{Compute} $\omega_B$ and $\alpha_B$ when $t = \frac{\pi}{6}$
       seconds.
-  \end{itemize}
-\end{minipage}
-\hfill
-\begin{minipage}[t]{0.35\linewidth}
-  \minipageTopAnchor
-  \includegraphics[width=1.0\linewidth]{mayflower.jpg}
-  \\[1.5pc]
-  \includegraphics[width=1.0\linewidth]{boat.png}
+      %
+    \\[0.45pc]
+    \Solution{}{0.99\linewidth}{
+      The rolling constraint gives us one useful equation. We can get another
+      one by implicit differentiation:
+      $$\minus \dot{\omega_A} R_A \dot{\omega_B} R_B
+        \text{\symmetricSpace{1cm}{or}}
+        \minus\alpha_A R_A \alpha_B R_B$$
+      %
+      We are given $\omega_A = 10 \sin(t)$. Taking the derivative with respect
+      to time yields:
+      %
+      $$\dot{\omega_A} = 10 \cos(t)
+        \text{\symmetricSpace{1cm}{or}}
+        \alpha_A = 10 \cos(t)$$
+      %
+      So plugging in:
+      \\[0.45pc]
+      \begin{tabular}{r@{\equals[\;]}l}
+        $\omega_B (t = \frac{\pi}{6})$
+        &$\minus \frac{R_A}{R_B} \omega_A(t = \frac{\pi}{6})$
+        \\[0.5pc]
+        &$\minus \frac{0.3 m}{6 m} (10 \sin(\frac{\pi}{6}))$
+        \\[0.5pc]
+        &$\minus 0.25 \frac{rad}{s}$
+      \end{tabular}
+      \hspace{1cm}
+      \begin{tabular}{r@{\equals[\;]}l}
+        $\alpha_B (t = \frac{\pi}{6})$
+        &$\minus \frac{R_A}{R_B} \alpha_A(t = \frac{\pi}{6})$
+        \\[0.5pc]
+        &$\minus \frac{0.3 m}{6 m} (10 \cos(\frac{\pi}{6}))$
+        \\[0.5pc]
+        &$\minus 0.43 \frac{rad}{s^2}$
+      \end{tabular}}
```

Comment:
Put the brace that closes a macro always on the next line by itself. This makes it much easier to see where things end and reduces issues when copying and pasting things.